src/app/shared/validators/location-validate.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive } from '@angular/core';
+import { AbstractControl, NG_VALIDATORS, ValidationErrors, Validator, Validators } from '@angular/forms';
+import { ErrorCode } from '../components/field-error/errors';
+
+const LOCATION_REGEX = /[a-z]/gi;
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: '[locationValidator]',
```

Comment:
Czemu wyłączasz tę regułę? Lepiej byłoby się do niej zastosować albo wyłączyć globalnie, jeśli nie chcemy się do niej stosować. Teraz mamy ją włączoną i ciągle wyłączamy, trochę bez sensu.

Ta reguła ma o tyle sens, że dyrektywy z różnych paczek mogą ze sobą wchodzić w nieprzewidziane konflikty, więc lepiej trzymać się zasady, że wszystkie nasze powinny mieć ten sam prefiks.

Reply:
<img width="863" alt="image" src="https://user-images.githubusercontent.com/12514322/159445392-7e97513e-ba89-42be-a007-be4df901905b.png">
potrzebny ten app-? ja bym to wyłączył

Reply:
Wymaganie prefiksu jest bezpieczniejsze z powodów, które podałem. Jeśli chciałbyś to jednak wyłączyć, to zacznij wątek na kanale.

Reply:
OK, już zacząłem wątek.